sources/packages/web/src/components/common/StudentApplication.vue

```diff
@@ -269,6 +271,7 @@ export default {
     };
 
     const submitted = (args: any, form: any) => {
+      processing.value = true;
```

Comment:
when this property is set to false? what happens to the loading when there is an error in submission?

Reply:
Yes, you're right. I created a `processing` prop and get it from the parent component. Thanks!